package.json

```diff
@@ -21,6 +21,7 @@
     "reset": "run-s reset:git reset:node-modules reset:yarn-lock configure",
     "configure": "yarn run init-registries && yarn install --frozen-lockfile --non-interactive && yarn run build:dev:backend",
     "install-yarn": "npm install --global yarn@1.22.17",
+    "corepack": "npm i -g corepack && corepack enable && corepack prepare yarn@stable --activate",
```

Comment:
I have 2 questions here.
1. Should we not install a specific version of yarn (currently: 1.22.17)?
2. If we make yarn available via corepack, do we still need `install-yarn` script? (@petermetz what's your view on this?)

Reply:
@jagpreetsinghsasan 

> Should we not install a specific version of yarn (currently: 1.22.17)?

100% agreed. @charellesandig Please fix this as well, gotta pin it to an exact version like we do with everything else in general.

> If we make yarn available via corepack, do we still need install-yarn script? (@petermetz what's your view on this?)

I would still leave it in because the enablement of corepack is optional at the end of the day. If someone prefers to just manually installing yarn and doing it the old way, they should be able to.

With that said, once all the LTS NodeJS versions that don't have yarn enabled by default have been end-of-life'd, THEN we could remove the install-yarn script because no one will ever need it again (unless they are purposefully running on some extremely outdated version of NodeJS but I don't want to support that in any way TBH). This will be years from now though because the CorePack thing is relatively new and LTS goes 5 or 6 years I think.

cc: @charellesandig